tools/ncf_rudder.py

```diff
@@ -159,15 +159,22 @@ def write_rudder_reporting_file(path,technique):
   file.close()
 
 
+def escape_entities( str ):
+    str = str.replace("&", "&amp;")
+    str = str.replace("<", "&lt;")
+    str = str.replace(">", "&gt;")
+    str = str.replace("\"", "&quot;")
```

Comment:
It seems to be only a common convention (http://stackoverflow.com/questions/6800467/quotes-in-xml-single-or-double), but we indeed use only double quotes.